hooks/build

```diff
@@ -263,18 +309,23 @@ if [[ "${AIRFLOW_CONTAINER_USE_PULLED_IMAGES_CACHE}" == "true" ]]; then
     echo
     echo
     if [[ "${AIRFLOW_CONTAINER_FORCE_PULL_IMAGES}" == "true" ]]; then
-        echo "Force-pull base python image. This forces to get the latest version."
+        echo "Force-pull base python image. This forces to get the most recent versions from the repository."
         echo
         set -x
         docker pull "${PYTHON_BASE_IMAGE}"
         set +x
         echo
     fi
-    IMAGES_TO_PULL="${AIRFLOW_CI_IMAGE}"
-    if [[ "${AIRFLOW_CONTAINER_SKIP_MAIN_IMAGE}" == "true" ]]; then
-        echo "Skip downloading the main Airflow image"
+    IMAGES_TO_PULL=""
+    if [[ "${AIRFLOW_CONTAINER_SKIP_CI_IMAGE}" == "true" ]]; then
+        echo "Skip downloading the CI Airflow image"
     else
-        IMAGES_TO_PULL="${IMAGES_TO_PULL} ${AIRFLOW_IMAGE}"
+        IMAGES_TO_PULL="${IMAGES_TO_PULL} ${AIRFLOW_CI_IMAGE}"
+    fi
+    if [[ "${AIRFLOW_CONTAINER_SKIP_SLIM_CI_IMAGE}" == "true" ]]; then
+        echo "Skip downloading the slim Airflow image"
+    else
+        IMAGES_TO_PULL="${IMAGES_TO_PULL} ${AIRFLOW_SLIM_CI_IMAGE}"
```

Comment:
Where is `IMAGES_TO_PULL` being used? I can't find another place where the variable is being used.

Reply:
It's just two lines below. It was hidden by Github as it did not change :).